### PR TITLE
[IN-2569] Install exact version of DEN Agent

### PR DESCRIPTION
**Pull request by martonszasz-bitrise**

### CODE REVIEW

File: roles/bitrise-den/tasks/main.yml [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+- name: Get asset id list
+  get_url:
+    url: "https://api.github.com/repos/bitrise-io/den/releases"
+    dest: /var/tmp/bitrise-den-releases.json
+    headers:
+      Authorization: "token {{ lookup('env', 'GITHUB_ACCESS_TOKEN') }}"
+- name: Get asset id for selected version
+  shell: >
+    set -o pipefail && \
+      cat /var/tmp/bitrise-den-releases.json | /usr/local/bin/jq ". | map(select(.tag_name == \"{{ bitrise_den_version }}\"))[0].assets | map(select(.name | contains(\"darwin\")))[0].id"
```
Comment (fehersanyi-bitrise) [inline roles/bitrise-den/tasks/main.yml:16]: can you make this multiline as well? for better readability?

**Review by fehersanyi-bitrise — DISMISSED** [review]

**Comment by martonszasz-bitrise** [comment]
My previous round failed on a seemingly simple step:
```
- name: Make sure jq is installed
  homebrew:
    name: "jq"
    state: present
    update_homebrew: true
```

```
    bitrise-den : Make sure jq is installed ------------------------------- 604.80s
```

Any ideas would be welcome.

**Review by bitrise-tom — APPROVED** [review]

**Review by fehersanyi-bitrise — APPROVED** [review]
lgtm

**Comment by arpadkun** [comment]
Congrats and thanks for the first PR! @martonszasz-bitrise  :)

**Pull request merged**